common/taskana-common/src/main/resources/sql/h2/taskana_schema_update_7.0.0_to_7.1.0_h2.sql

```diff
@@ -0,0 +1,20 @@
+-- this script updates the TASKANA database schema from version 7.0.0 to version 7.1.0.
+SET SCHEMA %schemaName%;
+
+INSERT INTO TASKANA_SCHEMA_VERSION (ID, VERSION, CREATED)
+VALUES (nextval('TASKANA_SCHEMA_VERSION_ID_SEQ'), '7.1.0', CURRENT_TIMESTAMP);
+
+ALTER TABLE TASK
+    ADD COLUMN NUMBER_OF_COMMENTS INT DEFAULT 0;
+
+UPDATE TASK t
+SET NUMBER_OF_COMMENTS = (
+	SELECT COUNT(tc.ID)
+	FROM TASK_COMMENT tc
+	WHERE tc.TASK_ID = t.ID
+)
+WHERE EXISTS (
```

Comment:
It doesn't matter in the end but why didn't you change this query to right outer join as well?

Reply:
because h2 doesn't have the update.. set.. from.. syntax, so it didn't work on h2 using the right outer join